apps/server/src/modules/server/server.module.ts

```diff
@@ -85,7 +85,11 @@ export const defaultMikroOrmOptions: MikroOrmModuleSyncOptions = {
 		new NotFoundException(`The requested ${entityName}: ${where} has not been found.`),
 };
 
-const setupSessions = (consumer: MiddlewareConsumer, redisClient: RedisClient | undefined, logger: LegacyLogger) => {
+export const setupSessions = (
+	consumer: MiddlewareConsumer,
+	redisClient: RedisClient | undefined,
+	logger: LegacyLogger
+) => {
 	const sessionDuration: number = Configuration.get('SESSION__EXPIRES_SECONDS') as number;
```

Comment:
I think it is cleaner to put it to the server config file and use it in this file.

Reply:
When I moved it to server.config.ts then  code coverage doesn't cover these 2 lines	
const RedisStore: connectRedis.RedisStore = connectRedis(session);
		store = new RedisStore({

Reply:
done